src/utils/sortIdentities.ts

```diff
@@ -0,0 +1,23 @@
+import { ProviderUser } from "src/@types/ProviderUser";
+
+import { IdentityTypes, Identity } from "../@types/Identity";
+import { SortedIdentities } from "../@types/SortedIdentities";
+
+export function sortIdentities(fetchedData: {
+  data?: { provider: ProviderUser } | null;
+}): SortedIdentities {
+  const phoneIdentities: Identity[] = [];
+  const emailIdentities: Identity[] = [];
+  if (fetchedData.data) {
+    const identities: Identity[] = fetchedData.data.provider.user.identities;
+    identities.forEach((identity) => {
+      if (identity.type === IdentityTypes.EMAIL.toLowerCase()) {
+        emailIdentities.push(identity);
+      } else if (identity.type === IdentityTypes.PHONE.toLowerCase()) {
+        phoneIdentities.push(identity);
```

Comment:
You can remove the lower casing now 😁.

Reply:
I think this is the only place where we do this, but you can double-check just to be sure.

Reply:
I did double-check and they were no other place where IdentityTypes were lower cased, it's fixed for this file 074d6a9

Reply:
Perfect! Good job mate!